test/image/mocks/gl3d_marker_symbols.json

```diff
@@ -0,0 +1,77 @@
+{
+  "data": [
+    {
+      "type": "scatter3d",
+      "x": [1, 2, 3, 4, 5, 6, 7, 8],
+      "y": [1, 1, 1, 1, 1, 1, 1, 1],
+      "z": [1, 1, 1, 1, 1, 1, 1, 1],
+      "mode": "markers",
+      "text": [
+        "marker symbol: 'circle'",
+        "marker symbol: 'circle-open'",
+        "marker symbol: 'square'",
+        "marker symbol: 'square-open'",
+        "marker symbol: 'diamond'",
+        "marker symbol: 'diamond-open'",
+        "marker symbol: 'cross'"
+      ],
+      "marker": {
```

Comment:
We should make sure new marker symbols are positioned and scaled properly in gl3d before adding them. To be completed in a future PR.